csaf_2.1/prose/edit/src/conformance.md

```diff
@@ -685,13 +686,16 @@ Secondly, the program fulfills the following for all items of:
   set the value to `multiplier`.
 * `/document/title`: If the value contains the `/document/tracking/id`, the CSAF 2.0 to CSAF 2.1 converter MUST remove the `/document/tracking/id`
   from the `/document/title`. In addition, separating characters including but not limited to whitespace, colon, dash and brackets MUST be removed.
-* `/vulnerabilities[]/cwes[]`: The CSAF 2.0 to CSAF 2.1 converter MUST determine the CWE specification version the given CWE was selected from by
-  using the latest version that matches the `id` and `name` exactly and was published prior to the value of `/document/tracking/current_release_date`
-  of the source document. If no such version exist, the first matching version published after the value of `/document/tracking/current_release_date`
-  of the source document SHOULD be used.
-  > This is done to create a deterministic conversion.
+* `/vulnerabilities[]/cwes[]`:
+  * The CSAF 2.0 to CSAF 2.1 converter MUST remove all preceding and trailing white space from the `name`.
```

Comment:
"white space" --> "space characters" or "spaces"

Reply:
I was using the term "white space" as this a fixed term in [EMCA 262](https://262.ecma-international.org/15.0/index.html#sec-white-space) which is the standard for [JSON patterns](https://json-schema.org/understanding-json-schema/reference/regular_expressions).

Reply:
Now tracked as #995